Assets/MixedRealityToolkit.Examples/Demos/Audio/Scripts/obj.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
+guid: 1e2e3b9f4400da24da455b576ba53c6c
+folderAsset: yes
+DefaultImporter:
```

Comment:
Is this meta file intentional? It looks like an artifact from a generated "obj" folder which was then deleted?